README.md

```diff
@@ -460,6 +460,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
 
 ## Code Generation
 
+  * [DhiWise](https://www.dhiwise.com/) — Seamlessly turn Figma designs into dynamic Flutter & React applications with DhiWise's innovative code generation technology, optimizing your workflow and helping you craft exceptional mobile and web experiences faster than ever before.
```

Comment:
is this entirely free or do you pay for some features? Seems unlikely this would be entirely free.

Reply:
@ripienaar it's entirely free.